scripting.md

```diff
@@ -0,0 +1,104 @@
+# SuperTux Level Editor Scripting Tutorial
+
+## What is Scripting?
+
+Scripting in SuperTux is when a game object is altered using things like a script trigger or a switch.  Scripting is written in the Squirrel coding language.
+
+#### What sort of things use scripting?
+
+Scripts can be activated by the following things, plus many more:
+
+* Script Trigger, which activates the script.
+* Switch/Button, which activate scripts when used.
+* Death Scripts, which activate when an enemy dies.
+* Init-Scripts, which activate when Tux enters the sector.
+
+In order to edit a script, right click a scriptable object and open the script editor.
+
+## Structure of scripts
+
+The basic structure of a line of script is as so:
+
+    object.action(Number);
+
+"object" is what you are going to activate, "action" is what the object does, and "Number" is the property of the action, like a number, series of numbers, or words. The semicolon activates the script, you have to use it in most situations.
+
+### Some scriptable objects
+
+Here are some things you can script in a sector:
+
+* Light: 
+
+      settings.set_ambient_light(color1, color2, color3);
+
+ where "color" is a decimal number between 0 and 1.
+
+* Music: 
+
+      play_music(\"music/song\"); 
+
+where "song" is a piece of music from the music folder.
+
+* A change to another sector's spawnpoint, or a spawnpoint in the current sector, is as shown: 
+
+      Level.spawn(\"sector\", \"spawn\");
+
+where "sector" is the sector and "spawn" is the spawnpoint.
+
+
+Are you starting to get it?
+
+### Cutscenes
```

Comment:
I don't think cutscenes are a basic concept and I doubt it should be explained here.

Reply:
i removed them.